modules/ROOT/pages/bolt/message.adoc

```diff
@@ -1534,6 +1559,18 @@ RECORD [{"point": [1, 2]}, "example_data", 123]
 
 The sections below list the changes compared to the previous version.
 
+[[messages-summary-52]]
+=== Version 5.2
+
+** `HELLO`, `BEGIN` and `RUN` messages now accept Notifications config.
```

Comment:
is it worth listing them explicitly, since they are two? Like

> ** `HELLO`, `BEGIN` and `RUN` messages now accept notifications options `notifications_minimum_severity` and `notifications_disabled_categories`.